authors/apps/authentication/serializers.py

```diff
@@ -182,3 +188,58 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+def email_structure(to_email, token):
+    user_info = {"user": {
+            "password": "new_password"}
+        }
+
+    # email setup
+    subject = 'Password reset'
+    message = """
+                    You're receiving this email because you invoked a password reset on
+                    Authors haven. If you think it is a mistake to receive this email just ignore it.
+                    ----- Use Postman to reset the email ----
```

Comment:
Postman is for developers and technical people. All a user needs to do is click the link.

Reply:
@inno-asiimwe  Thanks for that notice. Am going to handle this.